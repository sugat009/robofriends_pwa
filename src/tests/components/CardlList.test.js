import React from "react";
import {shallow} from "enzyme";

import CardList from "../../components/CardList";

it('should render CardList component', function () {
    const mockRobots = [
        {
            id: 1,
            name: "Jon Snow",
            username: "jonjon",
            email: "jon@gmail.com"
        }
    ];
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
