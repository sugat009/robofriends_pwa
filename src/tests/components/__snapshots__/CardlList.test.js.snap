// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CardList component 1`] = `
<div>
  <Card
    email="jon@gmail.com"
    id={1}
    key="0"
    name="Jon Snow"
  />
</div>
`;
